lib/image_processing/vips.rb

```diff
@@ -49,6 +49,14 @@ def resize_and_pad(image, width, height, gravity: "centre", extend: nil, backgro
         image.gravity(gravity, width, height, **embed_options)
       end
 
+      def rotate(image, degrees, **options)
+        if degrees % 90 == 0
+          image.rot(:"d#{degrees % 360}")
+        else
+          image.similarity(angle: degrees, **options)
+        end
+      end
```

Comment:
I would make `background: nil` an explicit option here, to avoid the undocumented functionality of being able to pass down any option to similarity (but only when the angle isn't 90°).

```ruby
# I'd use one of these

# alt1
image.similarity(angle: degrees, background: options[:background])

# alt 2
image.similarity(angle: degrees, **options.slice(:background))

# alt 3
def rotate(image, degrees, background: nil
  # …
  image.similarity(angle: degrees, background: background)
end